### ISSUE TITLE
About hyper-parameters lambda

### ISSUE THREAD

**Issue by codyshen0000** [pkuxmq/Invertible-Image-Rescaling#19]
Hi
In Equation 10 of the paper,  lambda_1 and lambda_3 are 1, but lambda_2 is set to 16. why setting lambda_2 so large?

**Comment by pkuxmq** [comment]
Hi, because we calculate the sum of losses on all pixels, so we simply set lambda_2 = s^2 (where s is the scale) to balance the weight for HR images and LR images.

**Comment by codyshen0000** [comment]
Thanks for your reply!

**Comment by codyshen0000** [comment]
Thanks for your reply!
 Since it  is used for balancing HR and LR,  I'm curious why not choose to increase the reconstruction weight of lambda_1?

**Comment by pkuxmq** [comment]
Hi, because HR images have s^2 times the number of pixels of LR images, so its loss is larger (we calculate the sum). If the reconstruction weight is much larger, there may be training instability at first. We simply keep their order to be equal (it is equivalent if we set lambda to be equal when we calculate the loss by mean over pixels). It is possible if there are better hyper-parameters.

**Comment by codyshen0000** [comment]
I see.  Thanks again!

**Comment by johnnylu305** [comment]
@pkuxmq 
Hi, may you explain why there may be training instability at first if the reconstruction weight is larger?

**Comment by johnnylu305** [comment]
> @pkuxmq
> Hi, can you explain why there may be training instability at first if the reconstruction weight is larger?
